shell/app-shell/app-shell.js

```diff
@@ -478,16 +464,6 @@ class AppShell extends Xen.Base {
       }
     });
   }
-  hotkey(key, e) {
```

Comment:
This method was designed to handle any hotkey we may want to implement. 

In the interest of YAGNI, I'm not averse to removing it completely, but make sure you remove the caller-chain too (`_initHotKeys` and it's caller in `_didMount`).

Reply:
Oops, didn't catch that, cleaned up.